content/en/security/code_security/static_analysis/custom_rules/guide.md

```diff
@@ -0,0 +1,744 @@
+---
+description: A complete guide to writing custom rules for Datadog.
+title: Static Code Analysis Custom Rule Guide
+---
+
+This guide builds on the [Static Analysis Custom Rules Tutorial][1] to give a full description of how to write custom rules, along with tips and tricks and common pitfalls to avoid.
```

Comment:
```suggestion
This guide builds on the [Static Analysis Custom Rules Tutorial][1], providing a full description of how to write custom rules, along with tips, tricks, and common pitfalls to avoid.
```